coa-helm/templates/cbas-api.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.cbas.coa-enabled }}
```

Comment:
Should we just make the `{{- if }}` block surround the whole file? Seems like we'd always want to control the deployment+service+configmap as a unit.

Reply:
Eh, don't feel that strongly

Reply:
Yeah the `---` essentially declares a new k8s resource, even though they are in the same yaml file -- e.g. there are 3 different k8s resources in that file